GLTFSDK.Test/CMakeLists.txt

```diff
@@ -16,6 +16,11 @@ if (MSVC)
     # Set warning level to 4 (/W4)
     target_compile_options(GLTFSDK.Test PRIVATE "/Zi;/W4;/EHsc")
 
+    # Use __stdcall calling convention for Win32/x86 builds.
+    if(CMAKE_VS_PLATFORM_NAME STREQUAL "Win32")
+        target_compile_options(GLTFSDK.Test PRIVATE "/Gz")
+    endif()
```

Comment:
I'm not sure what's going on with this. We just want to be able to handle different calling conventions. Is this to make sure the calling convention works? If so, it seems like something we should pass in from CI.